### DISCUSSION TITLE
Why does Prisma use `text` type in Postgres for `String` in Prisma Schema by default?

### DISCUSSION THREAD

**Discussion by talhabalaj** [prisma/prisma#9915] [Q&A]
Because it's `varchar(191)` MySQL. Isn't it generally a good idea to limit the size of the string?
I do know I can use `@db.Varchar(<length>)` to get desired length. I am just curious about the default choice being `text` in Posgres.

**Comment by ryands17** [comment]
@talhabalaj 👋 

It doesn't make a difference, actually using `text` gives a better performance. Have a look at [this](https://www.depesz.com/2010/03/02/charx-vs-varcharx-vs-varchar-vs-text/) article for an in-depth comparison.

**Reply by shtse8** [reply]
interesting, but seems wired. using text is performing better than varchar....

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
